Lesser Summon Bees

Paxilmaniac opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds a single spell to the wizard's arsenal, lesser summon bees, which summons... bees.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Ok so what if we took an extremely funny hostile mob and made it a spell for an extremely funny antag.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Lesser summon bees, a spell that summons bees
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

carda-sometimesknownas-paxilmaniac committed:
not the bees

oh god no

Commit:
Oh god let's not have it able to be instant though

that would be really bad

optimumtact commented:
nobody, not a single soul:
paxilmaniac: ADDS WIZARD STUFF LOL

wesoda25 commented:
Viscerators would be cooler...

GuyonBroadway commented:
I added this to hippie back when I used to have any involvement in that place, so well over a year ago.

Their version, rather than decrease the timer, summoned more bees per cast. 

This kind of spell ends up _bee_ing far weaker than you think unless you triple down on it and invest all your points into it. 


But you DO need to save some points for at least one rank in magic missile. 

https://youtu.be/xdkLZNyEMVU

Ratvar commented:
Lesser summon bees implies existence of forbidden arcana, Major summon bees.

ATH1909 commented on code/modules/spells/spell_types/wizard.dm:
```diff
@@ -229,6 +231,18 @@
+/obj/effect/proc_holder/spell/aoe_turf/conjure/creature/bee
+	name = "Lesser summon bees"
+	desc = "This spell magically kicks a transdimensional beehive, instantly summoning a swarm of bees to your location."
+	charge_max = 1000
```
what about the charge_min?

ATH1909 commented on code/modules/spells/spell_types/wizard.dm:
```diff
@@ -229,6 +231,18 @@
+	summon_amt = 9
+	action_icon_state = "bee"
+
+	summon_type = /mob/living/simple_animal/hostile/poison/bees/toxin
```
give these bees a faction that matches that of the wizard that summons them, or else this is just going to be an elaborate suicide spell (unless you're an android, you're a podperson, or you're wearing a gem-encrusted hardsuit)

necromanceranne replied:
Alternatively, the wizard gains that faction upon purchase of the spell.

Paxilmaniac replied:
Bees simply just have the "hostile" faction, which means if I give the wizard that faction, every hostile mob will ignore them

ATH1909 replied:
add a bee faction and make bees (and podpeople) use that instead of the weird snowflakey reaction bees current have towards podpeople

Fikou replied:
> add a bee faction and make bees (and podpeople) use that instead of the weird snowflakey reaction bees current have towards podpeople

just use the plants faction

Kaffe-work commented:
based, more wiz stuff

Commit:
Bee

Makes it take less time to recharge, while still preventing instant summon bees

Paxilmaniac commented:
> Lesser summon bees implies existence of forbidden arcana, Major summon bees.

We don't talk about such dark magics here.

GuyonBroadway commented:
Adminspawn only spell. 

Bees start coming out every vent.

lolman360 commented:
greater summon bee

just a really big bee

Paxilmaniac commented:
Bruh

ExcessiveUseOfCobblestone commented:
it would make sense if the bees summoned would not attack the wizard.

Rohesie commented on code/modules/spells/spell_types/wizard.dm:
```diff
@@ -229,6 +231,19 @@
+	invocation = "NOT THE BEES"
+	summon_amt = 9
+	action_icon_state = "bee"
+	cooldown_min = 200
```
```suggestion
	cooldown_min = 20 SECONDS
```

Rohesie commented:
Not a big fan of the spam potential, but then again, it's wizard.

Commit:
Update code/modules/spells/spell_types/wizard.dm

What Rohesie said to do with the minimum cooldown

Co-authored-by: Rohesie <rohesie@gmail.com>

Paxilmaniac commented:
I accidentally deleted the branch for this at some point so I can't edit it
